t/rose-task-run/31-app-bunch/suite.rc

```diff
@@ -0,0 +1,39 @@
+#!jinja2
+[cylc]
+    UTC mode=True
+    [[event hooks]]
+        timeout handler=rose suite-hook --shutdown
+        timeout=1
```

Comment:
Was written to be stylistically consistent with the other rose-task-run tests. I can update to ISO style if you prefer?